src/mixins/payment.js

```diff
@@ -0,0 +1,47 @@
+export default {
+  data () {
+    return {
+      conversationViewTriggered: false,
+    }
+  },
+  methods: {
+    async viewConversationAfterSuccessfulPayment () {
+      const routeQuery = this.$route.query
+
+      if (!this.conversationViewTriggered && this.paymentActive) {
+        this.conversationViewTriggered = true
```

Comment:
You said `afterAuth` was triggered twice, so I’m afraid that payment could fail the second time if this boolean has been set to true the first time. Please refer to my other answer about `afterAuth`.